frontend/packages/console-shared/src/components/catalog/catalog-view/CatalogView.tsx

```diff
@@ -65,10 +66,20 @@ const CatalogView: React.FC<CatalogViewProps> = ({
   hideSidebar,
 }) => {
   const { t } = useTranslation();
+  const fireTelemetryEvent = useTelemetry();
   const queryParams = useQueryParams();
   const activeCategoryId = queryParams.get(CatalogQueryParams.CATEGORY) ?? ALL_CATEGORY;
   const activeSearchKeyword = queryParams.get(CatalogQueryParams.KEYWORD) ?? '';
   const activeGrouping = queryParams.get(CatalogQueryParams.GROUPING) ?? NO_GROUPING;
+  const activeCategoryType = queryParams.get(CatalogQueryParams.TYPE) ?? '';
+
+  React.useEffect(() => {
+    fireTelemetryEvent('Developer Catalog', {
+      categories: activeCategoryId,
+      type: activeCategoryType,
+    });
```

Comment:
Do we need events for data that can captured via URL already?

Reply:
URL change is already being captured here https://github.com/openshift/console/blob/master/frontend/public/components/app.jsx#L288 

so am confused is it enough in this case or still need to capture separately?

Reply:
URL events and the query string is captured already. Therefore we can already query for catalog filters in use